libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -769,47 +773,49 @@ def get_flash_pages_sizes(self):
-                         128, 128, 128, 128 ]
+                return [16, 16, 16, 16, 64, 128, 128, 128, 128, 128, 128, 128]
             elif self.get_config('FLASH_SIZE_KB', type=int) == 2048:
-                return [ 16, 16, 16, 16, 64, 128, 128, 128,
-                         128, 128, 128, 128,
-                         128, 128, 128, 128,
-                         128, 128, 128, 128,
-                         128, 128, 128, 128 ]
+                return [
+                    16, 16, 16, 16,
+                    64, 128, 128, 128,
+                    128, 128, 128, 128,
+                    128, 128, 128, 128,
+                    128, 128, 128, 128,
+                    128, 128, 128, 128
+                ]
             else:
                 raise Exception("Unsupported flash size %u" % self.get_config('FLASH_SIZE_KB', type=int))
         elif mcu_series.startswith('STM32F7'):
             if self.get_config('FLASH_SIZE_KB', type=int) == 512:
-                return [ 16, 16, 16, 16, 64, 128, 128, 128 ]
+                return [16, 16, 16, 16, 64, 128, 128, 128]
             elif self.get_config('FLASH_SIZE_KB', type=int) == 1024:
-                return [ 32, 32, 32, 32, 128, 256, 256, 256 ]
+                return [32, 32, 32, 32, 128, 256, 256, 256]
             elif self.get_config('FLASH_SIZE_KB', type=int) == 2048:
-                return [ 32, 32, 32, 32, 128, 256, 256, 256,
-                         256, 256, 256, 256]
+                return [32, 32, 32, 32, 128, 256, 256, 256,
+                        256, 256, 256, 256]
             else:
                 raise Exception("Unsupported flash size %u" % self.get_config('FLASH_SIZE_KB', type=int))
         elif mcu_series.startswith('STM32H7'):
-            return [ 128 ] * (self.get_config('FLASH_SIZE_KB', type=int)//128)
+            return [128] * (self.get_config('FLASH_SIZE_KB', type=int)//128)
         elif mcu_series.startswith('STM32F100') or mcu_series.startswith('STM32F103'):
-            return [ 1 ] * self.get_config('FLASH_SIZE_KB', type=int)
+            return [1] * self.get_config('FLASH_SIZE_KB', type=int)
         elif mcu_series.startswith('STM32L4') and self.mcu_type.startswith('STM32L4R'):
             # STM32L4PLUS
-            return [ 4 ] * (self.get_config('FLASH_SIZE_KB', type=int)//4)
+            return [4] * (self.get_config('FLASH_SIZE_KB', type=int)//4)
         elif (mcu_series.startswith('STM32F105') or
               mcu_series.startswith('STM32F3') or
               mcu_series.startswith('STM32G4') or
               mcu_series.startswith('STM32L4')):
-            return [ 2 ] * (self.get_config('FLASH_SIZE_KB', type=int)//2)
+            return [2] * (self.get_config('FLASH_SIZE_KB', type=int)//2)
         else:
             raise Exception("Unsupported flash size MCU %s" % mcu_series)
 
     def get_flash_npages(self):
-        page_sizes = self.get_flash_pages_sizes()
+        pages = self.get_flash_pages_sizes()
```

Comment:
This looks like a bug-fix that changes behavior. What are the impacts of this?

Reply:
Method is never called.

flake8 caught this bug.